backend/de.metas.cucumber/src/test/java/de/metas/cucumber/stepdefs/dataImport/C_DataImport_StepDef.java

```diff
@@ -182,6 +182,16 @@ public void store_string_requestBody_in_context(@NonNull final DataTable dataTab
 		testContext.setRequestPayload(payload.replaceAll("null", ""));
 	}
 
+	@And("store String as requestBody in context")
+	public void store_string_as_requestBody_in_context(@NonNull final DataTable dataTable) throws IOException
+	{
+		final Map<String, String> row = dataTable.asMaps().get(0);
```

Comment:
```suggestion
		final Map<String, String> row = CollectionUtils.singleElement(dataTable.asMaps());
```
This way it's clear that no further rows should exist